controller_manager/src/controller_manager.cpp

```diff
@@ -3073,7 +3070,7 @@ controller_interface::return_type ControllerManager::update(
     }
     // TODO(v-lopez) we could cache this information
     // https://github.com/ros-controls/ros2_control/issues/153
```

Comment:
in fact, we are now caching this information right? 
```suggestion
```
The linked issue got closed some years ago.